docs/contributing/pull-requests/feature-branches.md

```diff
@@ -0,0 +1,51 @@
+# Long-lived Feature Branches
```

Comment:
Will this page ever be relevant to community contributors? What do you thinking about making it Bitwarden-only?